loops-and-arrays.js

```diff
@@ -21,6 +26,13 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+for (let i = 0; i < numbers.length; i++) {
+
+    numbers[i] *= 3;
+    console.log(numbers[i]);
+}
+
```

Comment:
De uitkomst die is gegeven is anders, maar jouw uitkomst werkt net zo goed!
`for (let i = 0; i < numbers.length; i++) {
  numbers[i] = numbers[i] * 3;`